parser.js

```diff
@@ -58,21 +59,37 @@ function parseForLatLong(domObj) {
     //scan DOM to collect the records/data
     //for each restaurant, get values of the relevant nodes to form an entry
     const allRestaurants = domObj.window.document.querySelectorAll("div.restaurantDetails");
+    let index = 1;
     for (let restaurant of allRestaurants) {
         let entry = {};
+
+        entry.id = index;
```

Comment:
Not sure how important the ID here is. Cos MySQL will add its own ID and probably should let the DB do its thing.

Reply:
Will resolve in iteration 3. Thank you Mr Cheng.